woocommerce/woocommerce
# Fix/base address wording

### All Submissions:






### Changes proposed in this Pull Request:



Since this settings only sets country/state for the customer, we shouldn't really call it address, as it can be confusing, especially in context of some extensions, like Distant Rate Shipping.

### How to test the changes in this Pull Request:

1. Go to settings and check the new string is present in the Shipping section.
2. Check that there is no change in the tax section, as over there, we set the city and zip code as well.


### Other information:




### Changelog entry

> Update shipping base address settings wording.

### FOR PR REVIEWER ONLY:
